tests/integration/targets/ec2_vpc_peer/tasks/main.yml

```diff
@@ -65,41 +65,60 @@
     set_fact:
       connection_name: 'Peering connection for VPC {{ vpc_1 }} to VPC {{ vpc_2 }}'
 
-  - name: Create local account VPC peering Connection
+  - name: Create local account VPC peering Connection request
     ec2_vpc_peer:
       vpc_id: '{{ vpc_1 }}'
       peer_vpc_id: '{{ vpc_2 }}'
       state: present
       tags:
-        Name: 'Peering connection for VPC {{ vpc_1 }} to VPC {{ vpc_2 }}'
+        Name: '{{ connection_name }}'
     register: vpc_peer
+
   - name: Assert success
     assert:
       that:
         - vpc_peer is changed
         - vpc_peer is successful
         - "'peering_id' in vpc_peer"
+        - vpc_peer.vpc_peering_connection.requester_vpc_info.cidr_block == vpc_1_cidr
         - vpc_peer.peering_id.startswith('pcx-')
 
   - name: Store Connection ID
     set_fact:
       peer_id_1: '{{ vpc_peer.peering_id }}'
 
-  - name: (re-) Create local account VPC peering Connection (idempotency)
+  - name: (re-) Create local account VPC peering Connection request (idempotency)
     ec2_vpc_peer:
       vpc_id: '{{ vpc_1 }}'
       peer_vpc_id: '{{ vpc_2 }}'
       state: present
       tags:
         Name: '{{ connection_name }}'
     register: vpc_peer
+
   - name: Assert success
     assert:
       that:
         - vpc_peer is not changed
         - vpc_peer is successful
         - vpc_peer.peering_id == peer_id_1
 
+  - name: Accespt local VPC peering connection from destination VPC
```

Comment:
```suggestion
  - name: Accept local VPC peering connection from destination VPC
```

Reply:
you always a few seconds too fast, just fixed it locally already ;)

Reply:
Seems there was already an accept test further down, so I deleted mine again. So now we should be all good.